src/lib/components/WalletModal.svelte

```diff
@@ -12,24 +31,20 @@
   <header>
     <h1>Choose your wallet</h1>
   </header>
-  <div class="wallet">
-    <button id="blocto" on:click={() => configureFCLAndLogin("blocto")}>
-      <img src="/blocto-logo.jpg" alt="blocto logo" />
-      <span>Connect Blocto</span>
-    </button>
-  </div>
   <div class="wallet">
     <button id="dapper" on:click={() => configureFCLAndLogin("dapper")}>
       <img src="/dapper-logo.png" alt="dapper logo" />
       <span>Connect Dapper</span>
     </button>
   </div>
-  <div class="wallet">
-    <button id="other" on:click={() => configureFCLAndLogin("other")}>
-      <img src="/lilico-logo.jpg" alt="lilico logo" />
-      <span>Connect Lilico</span>
-    </button>
-  </div>
+  {#each services as service}
```

Comment:
This is much more elegant than hardcoding wallets, BUT, it causes the issue of showing only Dapper wallet for 2-3 seconds before the rest of the wallets show up. I suggest we either preload all wallets on page load, OR show loading indicator for ALL wallets until services are loaded.

Reply:
Per favore scusi! Il mio Svelte non è molto buono!
Battled that buffer issue again today. Think we'll need to sync unless you had any luck upgrading FCL to latest